core/broker/config.go

```diff
@@ -47,6 +48,7 @@ func NewDefaultConfig() Config {
 		File: FileConfig{
 			Enabled: false,
 		},
+		EventBusClientBufferSize: 100000,
```

Comment:
I've made the default for core bigger than that for the datanode,  10,000 events per sec perfectly reasonable, so even with buffer etc we shoujld allow some headroom (GC pause whatever) -  maybe the default should be bigger?